torchvision/transforms.py

```diff
@@ -43,7 +43,10 @@ class ToTensor(object):
     def __call__(self, pic):
         if isinstance(pic, np.ndarray):
             # handle numpy array
-            img = torch.from_numpy(pic.transpose((2, 0, 1)))
+            if len(pic.shape) >= 3:
+                img = torch.from_numpy(pic.transpose((2, 0, 1)))
+            else:
+                img = torch.from_numpy(pic.reshape((1,) + pic.shape))
```

Comment:
nit: might be simpler to do `pic[np.newaxis]` to append a new dimension into the array